spec/widgets/widget-view-factory.spec.js

```diff
@@ -55,6 +55,7 @@ describe('widgets/widget-view-factory', function () {
       this.widget = new cdb.core.Model({}, {
         layer: this.layer
       });
+      this.widget.dataviewModel = {};
```

Comment:
Not need anything else?

Reply:
No, just needed because of `dataviewModel._totals` would be `undefined.totals` otherwise